.github/patch-initial-checklist.md

```diff
@@ -24,30 +30,41 @@ The release pull request has been created! This checklist is a guide to follow f
   * Confidence check - check blocks are available and function.
   * Test to confirm new features/fixes are working correctly.
   * Smoke test – test a cross section of core functionality.
+
+## Update Pull Request description and get approvals
+
 * [ ] Go through the description of the release pull request and edit it to update all the sections and checklist instructions there.
 * [ ] Ask a team member to review the changes in the release pull request and for anyone who has done testing that they approve the pull request.
 * [ ] Execute `npm run deploy`
   * Note: the script automatically updates version numbers (commits on your behalf).
   * **ALERT**: This script will ask you if this release will be deployed to WordPress.org. You should only answer yes for this release **if it's the latest release and you want to deploy to WordPress.org**. Otherwise, answer no. If you answer yes, you will get asked additional verification by the `npm run deploy` script about deploying a patch release to WordPress.org.
-* [ ] If this is released to WordPress.org, edit the GitHub release and copy changelog into the release notes.
-* [ ] If this is deployed to WordPress.org, the `#team-rubik` slack instance will be notified about the progress with the WordPress.org deploy. Watch for that. If anything goes wrong, an error will be reported and you can followup via the GitHub actions tab and the log for that workflow.
-* [ ] If this is deployed to WordPress.org, after the wp.org workflow completes, confirm the following
+
+## If this release is deployed to WordPress.org...
+
+* [ ] Edit the GitHub release and copy changelog into the release notes.
```

Comment:
This is referring to https://github.com/woocommerce/woocommerce-gutenberg-products-block/releases, right? Could we maybe add a link to avoid confusion?
```suggestion
* [ ] Edit the [GitHub release](https://github.com/woocommerce/woocommerce-gutenberg-products-block/releases) and copy changelog into the release notes.
```

Also, I see this step is under _If this release is deployed to WordPress.org..._, but wondering if it would make our lives easier that all versions (no matter if they aren't deployed to .org) had a changelog in the releases page.

Reply:
> Also, I see this step is under If this release is deployed to WordPress.org..., but wondering if it would make our lives easier that all versions (no matter if they aren't deployed to .org) had a changelog in the releases page.

On the surface it appears like it'd make our lives easier, but typically we don't merge patch releases back to the main branch because it's all cherry-picked work. So, effectively this would _create_ work because we'd have to always merge back to the main branch to keep that changelog between releases.

Also, doing this comes at the cost of potentially introducing confusion for feature plugin users that look at the changelog and wonder why they didn't see the patch release in wp.org. I think it's better to keep the changelog in the `readme.txt` synced with what is explicitly released in the plugin.